pkg/config/autodiscover/testdata/pods_with_json.json

```diff
@@ -0,0 +1,1905 @@
+{
```

Comment:
This file does not belong to the "autodiscover" package. You should create a "testdata" folder inside the **networking** test suite folder and put it there, because these UTs are testing functions of the networking package. Also, it shouldn't have the full pod/s object: only the ports array, because that's the content of the "res" variable that the declarePortList function will use to call the parseVariables one:
```
[
    {
        "containerPort": 8080,
        "name": "http-probe",
        "protocol": "TCP"
    }
]
```
The file name should be something like "test_ports.json" instead.

Reply:
I was add 2 files: one for test for declarePort and one for test listeningPort.

Reply:
The name of file for "declarePort" is "test_ports.json" and the name of file for "listeningPort" is "test_listening_port.json". is Ok? @greyerof

Reply:
Yes, that name is ok for me.